test/util/TestPathOptions.cpp

```diff
@@ -0,0 +1,66 @@
+#include <boost/test/unit_test.hpp>
+
+#include <boost/filesystem/path.hpp>
+
+#include "util/OptionsDB.h"
+#include "util/i18n.h"
+#include "util/XMLDoc.h"
+
+BOOST_AUTO_TEST_SUITE(TestPathOptions);
```

Comment:
can probably make the converters sufficiently constexpr to allow compile-time testing as well